### PR TITLE
Code Jam 8 Site Updates - v2

### PR DESCRIPTION
**Pull request by janine9vn**
This PR adds some additional changes required before we can make the code jam 8 announcement.
Summary of changes:
- Updated Import Dates to reflect that the announcement is going out today
- Changed main NavBar to have one Events tab that should bring you to the main event page
- Updated wording about joining the Discord server
- Correctly spelled GitHub, YouTube
- Added in DigitalOcean as a Sponsor
- Clarified and updated the rules
- Ensured consistency for "Approved Frameworks"
- Added documentation links for the approved frameworks

### CODE REVIEW

File: pydis_site/templates/events/pages/code-jams/8/_index.html [hunk 1]
```diff
@@ -20,18 +20,26 @@
...
         <li>Wednesday, June 30 - The Qualifier closes</li>
-        <li>Friday, July 9 - Code Jam Begins</li>
+        <li>Friday, July 9 - Code Jam begins</li>
         <li>Friday, July 16 - Coding portion of the jam ends</li>
         <li>Sunday, July 18 - Code Jam submissions are closed</li>
     </ul>
+
+    <h3 id="technology"><a href="#technology">Technology</a></h3>
+    <p>
+        The chosen technology/tech stack for this year is <strong> Text User Interfaces</strong> (TUIs).
```
Comment (jb3) [inline pydis_site/templates/events/pages/code-jams/8/_index.html]:
```suggestion
        The chosen technology/tech stack for this year is <strong>Text User Interfaces</strong> (TUIs).
```

File: pydis_site/templates/events/pages/code-jams/8/_index.html [hunk 2]
```diff
@@ -20,18 +20,26 @@
...
+        Each team must create a program with one of <a>the approved frameworks</a> that creates a user interface that is text based.
```
Comment (jb3) [inline pydis_site/templates/events/pages/code-jams/8/_index.html]: Missing the link in the anchor here

```suggestion
        Each team must create a program with one of <a href="{% url "events:page" path="code-jams/8/frameworks" %}">the approved frameworks</a> that creates a user interface that is text based.
```

**Review by jb3 — CHANGES_REQUESTED** [review]

**Review by jb3 — APPROVED** [review]

File: pydis_site/templates/events/pages/code-jams/8/_index.html (hunk 1 above)
Comment (janine9vn) [reply pydis_site/templates/events/pages/code-jams/8/_index.html]: I thought I checked it last night, but isn't this adding a double space now?

Comment (jb3) [reply pydis_site/templates/events/pages/code-jams/8/_index.html]: No, this removes a double space.
Screenshot 2021-06-15 at 16 05 34

File: pydis_site/templates/events/sidebar/code-jams/8.html [hunk 3]
```diff
@@ -9,7 +9,7 @@
     <img src="{% static "images/events/summer_code_jam_2021.png" %}" alt="Summer Code Jam 2021">
     <h4 class="menu-label">Our Sponsors</h4>
     <a href="https://www.djangoproject.com/" target="_blank">
```
Comment (ToxicKidz) [inline pydis_site/templates/events/sidebar/code-jams/8.html]:
```suggestion
    <a href="https://www.digitalocean.com/" target="_blank">
```

**Review by ToxicKidz — CHANGES_REQUESTED** [review]
Just one change

**Review by Shivansh-007 — APPROVED** [review]

**Review by ToxicKidz — APPROVED** [review]
:shipit:

**Review by HassanAbouelela — APPROVED** [review]

**Pull request merged**